Files/consultations/routes.py

```diff
@@ -47,14 +45,9 @@ def NewConsultation():
     if request.is_json:
         seller_id = get_jwt_identity()  
         res = request.get_json()
-        if res['seller_id'] == seller_id:
-            response = AddConsultation(**res)
-    
-            return response, response.status_code
-        else:
-            response = jsonify({"message": "You are not authorized to add this consultation"})
```

Comment:
Why is this error removed?

Reply:
The error is handled in the utils functions

Reply:
That's better, nice!